### DISCUSSION TITLE
Request: Click row, call up details from table (in modal or section)

### DISCUSSION THREAD

**Discussion by danielrosehill** [Budibase/budibase#13976] [Help] [answered]
Hello Budibase people!

Trying to get this to work and thought I would see if anyone can pinpoint what I'm doing wrong!

Data table that should be publicly accessible:

![image](https://github.com/Budibase/budibase/assets/40233203/ee62070f-3608-480d-9364-b72de9d52bff)

The desired behavior is:

-> User clicks on a row
-> User gets taken to a popup modal (or a section below the table)

Configuration I set was to attempt to capture the row ID like this:

![image](https://github.com/Budibase/budibase/assets/40233203/2eb874dc-6b3c-4b5b-aa27-89fd33c3fdcc)

For display, I created this form block beneath that table:

![image](https://github.com/Budibase/budibase/assets/40233203/2587f19d-ec62-4551-9150-8677f862f75d)

And then tried to pass the Row ID like this:

![image](https://github.com/Budibase/budibase/assets/40233203/a5b554b1-1117-4525-970f-d89d872bca1c)

The preview pane is blank and the form doesn't populate.

Can anyone identify what I need to do differently?

**Comment by mikesealey** [comment, marked as answer]
This seems to be a duplicate of https://github.com/Budibase/budibase/discussions/13975 where I'm left some pointers, so I'll mark this one as closed. Hopefully everything you need is in the answer on the other post.

**Reply by danielrosehill** [reply]
@mikesealey - yes. Apologies. Thought I had closed this one off. My apologies. Thank you very much for taking the time to respond. Much appreciated!
